### ISSUE TITLE
Error 429, proxy won't fix

### ISSUE THREAD

**Issue by yxwu** [fent/node-ytdl-core#670]
I have Error 429 from days ago (I use ytdl-core from server side only), and I tried to use proxy, but it still doesn't work.

my question is: 
1. Is it possible that both my VPS ip and the proxy ip have been banned? Or google use any other way to identify?
2. Any solution?

Best

**Comment by Roki100** [comment]
1. yes it is possible
2. try using cookies from example of cookies

**Comment by yxwu** [comment]
Thanks

**Comment by javaarchive** [comment]
I heard from the youtube-dl issues, that sometimes you can get unblocked if you use ipv4

**Comment by fent** [comment]
duep of https://github.com/fent/node-ytdl-core/issues/635
